src/components/sections/secondary-banner.json

```diff
@@ -0,0 +1,55 @@
+{
+  "collectionName": "components_sections_secondary_banners",
+  "info": {
+    "displayName": "secondaryBanner",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "name": {
+      "type": "string",
+      "unique": false,
+      "maxLength": 256,
+      "required": true
+    },
+    "title": {
+      "type": "component",
+      "repeatable": false,
+      "component": "shared.heading"
+    },
+    "text": {
+      "type": "text"
+    },
+    "backgroundColor": {
+      "type": "enumeration",
+      "enum": [
+        "white",
+        "black",
+        "background",
+        "primary",
+        "secondary",
+        "tertiary"
+      ],
+      "required": true,
+      "default": "primary"
+    },
+    "image": {
+      "type": "media",
+      "multiple": false,
+      "required": true,
+      "allowedTypes": [
+        "images"
+      ]
+    },
+    "CTA1": {
```

Comment:
Suggestion:
I would rename `CTA1` to primary and `CTA2` to secondary.
If you don't like it let's have lowercase for current keys at least.

Reply:
Updated